src/app/core/services/user-profile.service.js

```diff
@@ -23,20 +23,21 @@ export default function (app) {
       };
 
       this.createFormInfo = function () {
-        const { firstName, lastName, role, phone, email, ava } = $rootScope.currentUser;
-        return { firstName, lastName, role, phone, email, ava };
+        const {uid, firstName, lastName, role, phone, email, ava } = $rootScope.currentUser;
+        return { uid, firstName, lastName, role, phone, email, ava };
       };
 
       this.saveToCurrentUser = function (data) {
         const { 
+          uid,
           firstName = '', 
           lastName = '', 
           role = usersMocksService.userRole, 
           phone = '', 
           email = '', 
           ava = '' } = data;
 
-        $rootScope.currentUser = { firstName, lastName, role, phone, email, ava };
+        $rootScope.currentUser = { uid, firstName, lastName, role, phone, email, ava };
       };
```

Comment:
Please, replace all this method `saveToCurrentUser` with the following one.
I forgot to do it last time. And it does the same but shorter

```
this.saveToCurrentUser = function (data) {
        $rootScope.currentUser = Object.assign($rootScope.currentUser, data);
      };
```